code/src/OSFramework/DrawingTools/ITool.ts

```diff
@@ -16,7 +18,8 @@ namespace OSFramework.DrawingTools {
          * Adds the completedElement (completemarker, completepolyline, etc.) event listeners to the correspondent element
          * The new handlers will create the shape/markers elements and remove the overlay created by the drawing tool on the map
          */
-        addCompletedEvent(): void;
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        addCompletedEvent(e?: any): void;
```

Comment:
Can you please add the explanation for the optional e parameter? I know it is only for the Leaflet provider. But let's make it clear and also easy to understand. Remember: the comment was only for the Google Provider.